3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1019,6 +1019,38 @@ MAIUSCOLO:
 ===== Glossario
 Tutte le occorrenze dei termini contenuti nel glossario sono evidenziati con una G in corsivo a pedice.
 
+===== Variabili Typst per termini ricorrenti
+
+Il gruppo ho predisposto una serie di comandi Typst per riferirsi a termini usati molto frequentemente. In particolare troviamo:
```

Comment:
Non si usano verbi in forma personale come "troviamo". Il documento deve essere in forma impersonale.

Cambiare in "I termini si riferiscono a: "